libia2/include/ia2.h

```diff
@@ -77,38 +77,42 @@ static uint32_t ia2_get_pkru() {
+          ":\n"                                                                \
+          "jmp " #function "\n")
 #else
-#define _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)    \
-    __asm__(".global ia2_sighandler_" #function "\n"  \
-            "ia2_sighandler_" #function ":\n"         \
-            "movq %rcx, %r10\n"                       \
-            "movq %rdx, %r11\n"                       \
-            "movq %rax, %r12\n"                       \
-            "xorl %ecx, %ecx\n"                       \
-            "xorl %edx, %edx\n"                       \
-            "mov_pkru_eax " #pkey "\n"                \
-            "wrpkru\n"                                \
-            "movq %r12, %rax\n"                       \
-            "movq %r11, %rdx\n"                       \
-            "movq %r10, %rcx\n"                       \
-            "jmp " #function "\n")
+#define _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)                             \
+  __asm__(".global ia2_sighandler_" #function                                  \
+          "\n"                                                                 \
+          "ia2_sighandler_" #function                                          \
+          ":\n"                                                                \
+          "movq %rcx, %r10\n"                                                  \
+          "movq %rdx, %r11\n"                                                  \
+          "movq %rax, %r12\n"                                                  \
+          "xorl %ecx, %ecx\n"                                                  \
+          "xorl %edx, %edx\n"                                                  \
+          "mov_pkru_eax " #pkey                                                \
+          "\n"                                                                 \
+          "wrpkru\n"                                                           \
+          "movq %r12, %rax\n"                                                  \
+          "movq %r11, %rdx\n"                                                  \
+          "movq %r10, %rcx\n"                                                  \
+          "jmp " #function "\n")
 #endif
 
-#define IA2_DEFINE_SIGACTION(function, pkey)                \
-    void ia2_sighandler_##function(int, siginfo_t*, void*); \
-    _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)
+#define IA2_DEFINE_SIGACTION(function, pkey)                                   \
+  void ia2_sighandler_##function(int, siginfo_t *, void *);                    \
+  _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)
 
-#define IA2_DEFINE_SIGHANDLER(function, pkey)  \
-    void ia2_sighandler_##function(int);       \
-    _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)
+#define IA2_DEFINE_SIGHANDLER(function, pkey)                                  \
+  void ia2_sighandler_##function(int);                                         \
+  _IA2_DEFINE_SIGNAL_HANDLER(function, pkey)
```

Comment:
Unnecessary formatting changes?

Reply:
This is also up to clang-format and arguably more consistent with the rest of the file. I can move the formatting to the bottom of the stack or a separate PR.